docker/Dockerfile.foxx

```diff
@@ -46,12 +53,12 @@ RUN mkdir -p ${DATAFED_DIR}
 RUN mkdir -p ${DATAFED_INSTALL_PATH}
 RUN mkdir -p ${DATAFED_DEFAULT_LOG_PATH}
 
-RUN curl -OL https://download.arangodb.com/arangodb311/DEBIAN/Release.key; apt-key add - < Release.key
+RUN curl -OL https://download.arangodb.com/arangodb312/DEBIAN/Release.key; apt-key add - < Release.key
 
 # RUN echo 'deb https://download.arangodb.com/arangodb311/DEBIAN/ /' | tee /etc/apt/sources.list.d/arangodb.list
 # If trust the site but the gpg key has expired can add [trusted=yes]
 # i.e.
-RUN echo 'deb [trusted=yes] https://download.arangodb.com/arangodb311/DEBIAN/ /' | tee /etc/apt/sources.list.d/arangodb.list
+RUN echo 'deb [trusted=yes] https://download.arangodb.com/arangodb312/DEBIAN/ /' | tee /etc/apt/sources.list.d/arangodb.list
 RUN apt update
```

Comment:
ArangoDB version should be defined in one place and used consistently everywhere else.